packages/extract-react-types/package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "extract-react-types",
   "version": "0.16.1",
-  "main": "src/index.js",
+  "main": "dist/extract-react-types.cjs.js",
   "repository": "atlassian/extract-react-types",
   "description": "Parse prop-types from react components using typescript or flow",
   "author": "James Kyle <me@thejameskyle.com>",
   "license": "Apache-2.0",
   "files": [
-    "src"
+    "dist"
```

Comment:
You still need `src` here so the flow types work when published because the .flow files point back to the source file.

Reply:
Good spot, I added it back